File: Functions/3.cpp
#include<bits/stdc++.h>
using namespace std;

int sum_of_digits(int n){
    int total = 0;
    int d;
    while(n>0){
        d = n%10;
        total+=d;
        n = n / 10;
    }
    return total;
}

int main(){
    cout<<sum_of_digits(14347);
}

File: Functions/3.py
def sum_of_digits(n):
    total = 0  
    while n > 0:
        d = n % 10 
        total += d  
        n = n // 10 
    return total

print(sum_of_digits(14347))  
print(sum_of_digits(1447))
print(sum_of_digits(1213))   